recipe/meta.yaml

```diff
@@ -1,48 +1,49 @@
 {% set name = "cadquery" %}
-{% set version = "1.2.5" %}
+{% set version = "2.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - folder: dist
-    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: b2ddd927cb81bc9ad2332692d4ab0b4cb36581543463b64bd2d388b3709ea0ae
-    patches:
-      - osx_freecad_path.patch
   - folder: src
-    url: https://github.com/dcowden/cadquery/archive/v{{ version }}.tar.gz
-    sha256: 75eb6f987aeaf3d86197353c53dc2808a78fa9f2ade651f5751e96aa14b23829
+    url: https://github.com/CadQuery/cadquery/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 9a71a3bb1185228f62b14810fe078489f4af61e02d9e2c683e68626ff9903dbc
 
 build:
   number: 0
   noarch: python
-  script: cd dist && python -m pip install --no-deps --ignore-installed .
+  script: cd src && python -m pip install --no-deps --ignore-requires-python --ignore-installed .
```

Comment:
can we remove the `--ignore-requires-python` since it's pinned below?

Reply:
also please add `pip install -vv` for more insight during the install

Reply:
It is required due to what is specced in setup.py